src/modules/test/main.c

```diff
@@ -0,0 +1,732 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <Common.h>
+#include <Module.h>
```

Comment:
Can also move Module.h to Common.h.

However, is Module.h really needed? If it's only main.c that includes it, all Module.h contents could be in main.c (and this could simplify the code even more)

Reply:
I kept it separate to make reusing this code in the platform easier in the future if/when C++ components are converted to C.